addons/binding/org.openhab.binding.jeelink/src/main/java/org/openhab/binding/jeelink/handler/AbstractJeeLinkConnection.java

```diff
@@ -0,0 +1,137 @@
+/**
+ * Copyright (c) 2010-2017 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.jeelink.handler;
```

Comment:
You should consider moving this (and many other classes) to an internal package like `org.openhab.binding.jeelink.internal`.